### ISSUE TITLE
GTA IV freezes in mission "Easy As Can Be" during in-game cutscene

### ISSUE THREAD

**Issue by Spoggi99** [ThirteenAG/GTAIV.EFLC.FusionFix#26]
Hello ThirteenAG,

I noticed a problem during the mission "Easy As Can Be" given by Brucie Kibbutz. After you reached the car you are supposed to steal, an in-game cutscene of Niko entering the car starts and the game freezes.

Up until now, I thought that the problem was isolated to me and related to other mods. However, I published a Steam guide called [How to make GTA IV: The Complete Edition perfectly playable in 2020](https://steamcommunity.com/sharedfiles/filedetails/?id=921826093) a few months ago (featuring FusionFix and Col.Accel among other fixes). This guide gained moderate popularity on Steam and I frequently have to answer comments and troubleshoot. One such comment described the exact same problem with "Easy As Can Be" that I've experienced. As a result, it seems like this freeze is a direct result of the FusionFix. The only way around it is to move the FusionFix out of the GTA IV directory, play the mission, save, and then reinsert the FusionFix afterwards.

I replayed the whole GTA IV story with your fix and "Easy As Can Be" is the only mission where it caused a game-breaking problem.

Thank you very much for your efforts and your great fixes - they bring joy to many of us GTA players, including myself! ;)

**Comment by ThirteenAG** [comment]
Can you try to disable various options and find out which one causes it? First disable all, and see if it happens that way.

**Comment by Spoggi99** [comment]
Sure, I will try that out as soon as I can.

Thank you for the quick reply!

**Comment by GoingToAruba** [comment]
Tested the mission while using the default FusionFix settings. Didn't experience any issues with the cutscene freezing on my end.

**Comment by Spoggi99** [comment]
Sorry for the very late reply. I was not able to replicate the problem and therefore assume that one of the latest builds of your mod fixed the problem.

Again, sorry for the late reply. Thank you for your incredible work on this mod!

**Comment by iYonga** [comment]
I tried removing FusionFix or limiting FPS to 30 or 50 none of those worked. Still crashes and hangs at that part

**Comment by ricmetal** [comment]
finally worked for me; setting cutsceneLimit in the .ini
thanks for your work, ThirteenAG!

**Comment by Aswanovic** [comment]
workaround when changed resolution to 60hz, fps limit to 30 and vsync after that switched to 60 fps cap again.

**Comment by mattfelten** [comment]
I'm having this freeze also. So far I've tried:
- ❌ Limiting FPS to 30
- ❌ Setting CutsceneFpsLimit to 50
- ❌ Setting CutsceneFpsLimit to 30
- ❌ Setting CutsceneFpsLimit to 30, FPS Limit to 30, and resolution to 60hz
- ❌ Renamed `GTAIV.EFLC.FusionFix.asi` to `GTAIV.EFLC.FusionFix.asi.bak` (hoping it would just uninstall the mod). It didn't forever freeze this time, it actually crashed.
- I tried to do ^ again with a CrashDumps folder created to get a log but this time it forever froze ...
